src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs

```diff
@@ -332,6 +332,34 @@ public List<string> GetPerformerScenes(string stashId)
             return scenes;
```

Comment:
Return a tuple with scenes and movies will save having to make another call since works contains both. After the change, you can get rid of the added method.

Reply:
GetPerformerScenes has been changed to GetPerformerWorks that returns a tuple/